snowpack/src/config.ts

```diff
@@ -509,7 +509,7 @@ function normalizeAlias(config: SnowpackConfig, createMountAlias: boolean) {
       replacement.startsWith('/')
     ) {
       delete cleanAlias[target];
-      cleanAlias[addTrailingSlash(target)] = addTrailingSlash(path.resolve(cwd, replacement));
```

Comment:
What problem is this change trying to fix? Please add a test to let others know what will be fixed by this change.

Reply:
> What problem is this change trying to fix? Please add a test to let others know what will be fixed by this change.

OK. I will add some tests soon.

Reply:
> What problem is this change trying to fix? Please add a test to let others know what will be fixed by this change.

well, I added a reproduce step.

Reply:
The reproduction step is helpful, but what if someone else touched this code. How would we know whether or not there was a regression with this fix?

If you could add your reproduction to `./tests/build/alias`, make sure it failed before, and your PR fixed it, then that would help us make sure this fix stays.

Reply:
Ok.  I will add some unit tests for the normalizeAlias function. actually, I want to use Cypress to do some test work instead of Jest. Can I involve Cypress in Snowpack? @drwpow @FredKSchott

Reply:
We already have an existing test setup with Jest that’s easy to add to and is relatively quick to run in CI, so please use what’s there already for this PR.

I’m not opposed to adding Cypress in the future, but I’d want that to be a separate discussion so we can evaluate what it costs in terms of slowing the test runner down, general testing strategy, flakiness, etc.

Reply:
Well，I  got it, I will do it as soon as possible.

Reply:
Some background: The reason for `addTrailingSlash` here is that we want consistent behavior. If the user gives us `"components": "./src/components"` or `"components/": "./src/components"`, we want to behave the same way. This is easier to tackle here than anywhere else.

So I'm -1 on removing the entirely. But, if we could replace it with `removeTrailingSlash` instead and still fix this bug, then there would be no safety guarentee lost.

Also, I'm still not sure I understand what bug this is solving. What breaks when you do `"components": "./src/components"` today?